charts/ndb-operator/templates/service-ndb-operator-webhook-service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app.kubernetes.io/component: webhook
+    app.kubernetes.io/created-by: ndb-operator
+    app.kubernetes.io/instance: webhook-service
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/name: service
+    app.kubernetes.io/part-of: ndb-operator
+  name: {{ include "ndb-operator.fullname" . }}-webhook-service
+  namespace: {{.Release.Namespace}}
+spec:
+  ports:
+  - port: 443
+    protocol: TCP
+    targetPort: 9443
+  selector:
+    control-plane: controller-manager
```

Comment:
I feel this needs to be templated too.

Reply:
The auto-generated text does not have namespace name "ndb-operator" in it, which implies this selector does not depend on the namespace name. I feel we should not add namespace context where k8s has not attached it.